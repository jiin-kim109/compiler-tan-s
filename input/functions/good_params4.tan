subr void holler(int r, int s) {
	print r \s s \n;
	
	for(n from r to s) {
		print n \s "Yoo hoo!!!!" \n;
	}
	return;
}

main {
	call holler( 4, 8 );
	
	var m := 0.34;
	call holler( 7, 10 );
	
	print "all done." \n;
}
