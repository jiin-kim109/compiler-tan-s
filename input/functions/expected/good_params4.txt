4 8
4 Yoo hoo!!!!
5 Yoo hoo!!!!
6 Yoo hoo!!!!
7 Yoo hoo!!!!
8 Yoo hoo!!!!
7 10
7 Yoo hoo!!!!
8 Yoo hoo!!!!
9 Yoo hoo!!!!
10 Yoo hoo!!!!
all done.
